### ISSUE TITLE
prepare_ensembl not working properly

### ISSUE THREAD

**Issue by niradsp** [ekawaler/pyQUILTS#14]
I have noticed two problems.

1) Given that ensembl is a one-based system, while refseq is zero-based, it appears that 1 needs to be subtracted from proteome and transcriptome start values.  I compared the proteome.bed file generated by prepare_ensembl to proteome.bed file generated by prepare_refseq.  So to give you an example, for TP53, if the proteome.bed file generated by refseq has start value of 768401, the ensembl file has start value of 7668402.  So I ran Quilts by subtracting one from the start values.  If I don't subtract one, there are close to 50 times more novel types being generated by quilts (and refseq had far fewer, close to 6000).

2)  Now I noticed  a very different problem.  This seems to be a reference genome issue.  I will give two examples to illustrate what I mean.   Both these examples are from Chromosome 19.
  a) The protein ID is ENSP00000496230 (transcript ID ENST00000591545) . The start location is 10984278 (actual location given  in the Mart database is 10984279, but I subtracted one).  I used the following code:

`library(Biostrings)`
`library(biomaRt)`
`dat<-readDNAStringSet('NiradData/Testing/chr19.fa.cmp1')`
`splitup<-unlist(strsplit(as.data.frame(dat)$x,''))`
`start=10984278`
`seq=splitup[(start):(start+95)]`
`seq=paste(seq,collapse='')`
`translate(DNAStringSet(seq)) `

The sequence above is translated correctly.  There were no issues.  It matches SMARCA4.

b) The IDs are transcript ID ENST00000595661 and  protein ID ENSP00000472808.  I am using the same code as above, but different start index:
`start=50031082`
`seq=splitup[(start):(start+9)]`
`seq=paste(seq,collapse='')`
`translate(DNAStringSet(seq))`

The above code is translating incorrectly.  There is a truncation after the second Amino Acid.  So I tried the following code:
`start=50031082`
`seq=splitup[(start+1):(start+9)]`
`seq=paste(seq,collapse='')`
`translate(DNAStringSet(seq))`

Here I added one to the start location.  This sequence is translating correctly.

To summarize, the transcript starting in location 10984278 did not require adding one, but the one starting in 50031082 required adding one.  The second example that I provided is being translated correctly by Quilts (even though in the example above it is not translating correctly).  The first example is not (in the above example it's the opposite).  What is causing this issue, and how may I fix it?  There are lots of sequences that were translated incorrectly, when I looked at the variant_proteome file.

I also tried downloading NCBI fasta file.  I noticed the same pattern.

**Comment by ekawaler** [comment]
Ensembl preparation should work better now
